homeassistant/bootstrap.py

```diff
@@ -207,6 +208,8 @@ def process_ha_core_config(hass, config):
         if key in config:
             setattr(hass.config, attr, config[key])
 
+    Entity.visibility.update(config.get('visibility', [{}])[0])
```

Comment:
Instead of having a list with 1 item it would be better to have the visibility key in the config file to be a hash.

```yaml
homeassistant:
    # control visibility of entity cards                                                                                                                                                                          
    visibility:                                                                                                                                                                                                   
        sensor.forecast_snow                     : hide  #     Forecast Snow                                                                                                                                      
        light.all_on_3                           : hide  #     All On                                                                                                                                             
        light.foyer_2                            : show  #     Foyer
```